test/BUILD.gn

```diff
@@ -1107,6 +1107,7 @@ test("brave_browser_tests") {
 
   if (!is_android) {
     sources += [
+      "//brave/browser/brave_autofill_browsertest.cc",
```

Comment:
do not add individual tests to `test/BUILD.g`, this should go in brave/browser/autofill and be a dep, but also do not add tests to the `!is_android` lists here, they belong in `android/android_browser_tests.gni`

Reply:
actually this PR made predate that change